docs/_includes/ds6/combobox.html

```diff
@@ -0,0 +1,166 @@
+
+    <h3>Default combobox</h3>
+    <p>By default, the first item should be selected if the user does not specify a selection. This matches the behavior of a native HTML select element.</p>
+
+    <div class="demo">
+        <div class="demo__inner" style="max-width: 200px;">
+            <span class="combobox">
+                <span class="combobox__control">
+                    <input type="text" value="Option 1" readonly role="combobox" aria-haspopup="combobox" aria-owns="combobox1" />
+                    <span class="combobox__arrow" aria-hidden="true"></span>
+                </span>
+                <div id="combobox1" class="combobox__options" role="listbox">
+                    <div class="combobox__option" role="option" aria-selected="true">
+                        <span>Option 1</span>
+                        <span class="combobox__status"></span>
+                    </div>
+                    <div class="combobox__option" role="option">
+                        <span>Option 2</span>
+                        <span class="combobox__status"></span>
+                    </div>
+                    <div class="combobox__option" role="option">
+                        <span>Option 3</span>
+                        <span class="combobox__status"></span>
+                    </div>
+                </div>
+                <select name="nativeOptions" hidden>
+                    <option value="item1">Option 1</option>
+                    <option value="item2">Option 2</option>
+                    <option value="item3">Option 3</option>
+                </select>
+            </span>
+        </div>
+        {% highlight html %}
+<span class="combobox">
+    <span class="combobox__control">
+        <input type="text" value="Option 1" readonly role="combobox" aria-haspopup="combobox" aria-owns="combobox1"/>
+        <span class="combobox__arrow" aria-hidden="true"></span>
+    </span>
+    <div id="combobox1" class="combobox__options" role="listbox">
+        <div class="combobox__option" role="option" aria-selected="true">
+            <span>Option 1</span>
+            <span class="combobox__status"></span>
+        </div>
+        <div class="combobox__option" role="option">
+            <span>Option 2</span>
+            <span class="combobox__status"></span>
+        </div>
+        <div class="combobox__option" role="option">
+            <span>Option 3</span>
+            <span class="combobox__status"></span>
+        </div>
+    </div>
+</span>
+        {% endhighlight %}
+    </div>
+
+    <h3>Selected combobox</h3>
+    <div class="demo">
+        <div class="demo__inner" style="max-width: 200px;">
+            <span class="combobox">
+                <span class="combobox__control">
+                    <input type="text" value="Option 2" readonly role="combobox" aria-haspopup="combobox" aria-owns="combobox2"/>
```

Comment:
Should be aria-haspopup="listbox"... like you had before :-)

Reply:
Oh, will fix.